src/js/components/Header/stories/typescript/Ref.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { Avatar, Anchor, Nav, Grommet, Header, Button } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const gravatarLink =
+  '//s.gravatar.com/avatar/b7fb138d53ba0f573212ccce38a7c43b?s=80';
+
+export const Ref = () => {
+  const ref = React.createRef<HTMLDivElement>();
+  const [color, setColor] = React.useState('');
+
+  const setRandomColor = () => {
+    const red = Math.floor(Math.random() * 255).toString(16);
```

Comment:
We should avoid random since it makes hard to have any sort of automated test for storybook.